drivers/telescope/lx200_10micron.cpp

```diff
@@ -771,7 +744,7 @@ bool LX200_10MICRON::SetTLEfromDatabase(int tleN)
     return 0;
 }
 
-bool LX200_10MICRON::CalculateTrajectory(int year, int month, int day, int hour, int minute, int nextpass, ln_date date_pass)
+bool LX200_10MICRON::CalculateSatTrajectory(char *start_pass_iso, char *end_pass_iso)
```

Comment:
good to see ISO8601 in use :) maybe name it start_pass_isodatetime for clarity ?

Reply:
Done